src/icons/general/IconSecurityClassification.js

```diff
@@ -0,0 +1,41 @@
+// @flow
+import * as React from 'react';
+import classNames from 'classnames';
+
+import AccessibleSVG from '../accessible-svg';
+import { bdlGray } from '../../styles/variables';
+
+type Props = {
+    className?: string,
+    color?: string,
+    height?: number,
+    strokeWidth?: number,
+    /** A text-only string describing the icon if it's not purely decorative for accessibility */
+    title?: string | React.Element<any>,
+    width?: number,
+};
+
+const IconSecurityClassification = ({
+    className = '',
+    height = 32,
+    color = bdlGray,
+    title,
+    strokeWidth = 2,
+    width = 32,
+}: Props) => {
+    const classes = classNames('bdl-IconSecurityClassification', className);
+
+    return (
+        <AccessibleSVG className={classes} height={height} title={title} viewBox="0 0 32 32" width={width}>
+            <g strokeWidth={strokeWidth} fill="none" fillRule="evenodd">
```

Comment:
You may be able to move the strokeWidth to the 2 paths below and get rid of the `<g>` wrapper completely

Reply:
`fill="none"` would also be needed by the 2 paths, would we avoid the `<g>` in the case ?

Reply:
sure, why not, less DOM